src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.NonAndroid.cs

```diff
@@ -387,6 +445,10 @@ private static bool TryLoadTzFile(string tzFilePath, [NotNullWhen(true)] ref byt
         /// 2. Get the default TZ from the device
         /// 3. Use UTC if all else fails.
         ///
+        /// On WASI / Browser
+        /// 0. if TZDIR is not set, use TZ variable as id to embedded database.
+        /// 1. fall back to unix behavior if TZDIR is set.
```

Comment:
nit, to be consistent with the other comments.

```suggestion
        /// 1. if TZDIR is not set, use TZ variable as id to embedded database.
        /// 2. fall back to unix behavior if TZDIR is set.
```